packages/deploy/deploy/500_marketplace/501_royalties_registry_set_ownership.ts

```diff
@@ -0,0 +1,25 @@
+import {DeployFunction} from 'hardhat-deploy/types';
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+
+const func: DeployFunction = async function (
+  hre: HardhatRuntimeEnvironment
+): Promise<void> {
+  const {deployments, getNamedAccounts} = hre;
+  const {execute, read, catchUnknownSigner} = deployments;
+  const {sandAdmin} = await getNamedAccounts();
+  const owner = await read('RoyaltiesRegistry', 'owner');
+  if (!(owner == sandAdmin)) {
+    await catchUnknownSigner(
+      execute(
+        'RoyaltiesRegistry',
+        {from: owner, log: true},
+        'transferOwnership',
+        sandAdmin
+      )
+    );
+  }
+};
+
+export default func;
+func.tags = ['RoyaltiesRegistry'];
```

Comment:
you need a specific tag to be able to call this script alone, ie
```
func.tags = ['Asset', 'AssetCreate', 'AssetCreate_setup'];
```

=> func.tags = ['RoyaltiesRegistry', 'RoyaltiesRegistry_setup'];